docker/nginx/conf.d/default.conf

```diff
@@ -57,16 +58,23 @@ server {
 
   # The request body is sent to the proxied server immediately as it is
   # received
-  proxy_request_buffering off;
+
+  # proxy_request_buffering off;
   # Sets the HTTP protocol v1.1 for proxying in order to not use the buffer
   # in case of chunked transfer encoding
-  proxy_http_version 1.1;
+
+  # proxy_http_version 1.1;
+
+  location = /ping {
```

Comment:
I think we should remove this from here and pass it below in the `location ~ ^/(login|`?

Reply:
I had troubles with the uwsgi and the ping that's why I put it there. I need to re test if this is fine.

Reply:
Changed it!